### ISSUE TITLE
Pin scenedetect < 0.6

### ISSUE THREAD

**Issue by Breakthrough** [dynamite-ready/movie-parser-cli#1]
Hi there!  I noticed you're using PySceneDetect, but haven't pinned the dependency.  v0.6 is slated to include some breaking changes to improve the API usage somewhat, so I would recommend that you pin your version to < 0.6, otherwise the current source code will break once that is released.

I also noticed that you created a StatsManager - I don't think you need that if you want to simplify things a bit.  Lastly, you may want to set the threshold in ContentDetector to 27 (the current default is a bit too high for most videos).  The default will be changed to this in v0.6.

-----------------------------

Reading through your source code was also very enlightening - many thanks for the amount of comments.  I also would very much like to improve the API, which is the primary goal for the immediate future releases.  The new quickstart for the v0.6 API is:
```
from scenedetect import open_video, SceneManager
from scenedetect.detectors import ContentDetector

video = open_video(path=video_path)
scene_manager = SceneManager()
scene_manager.add_detector(
    ContentDetector(threshold=threshold))
scene_manager.detect_scenes(video)
```

Hopefully you find that an improvement from the current design, so keep an eye out for v0.6 - happy to hear any other feedback you might have though.  I'll also look into adding a high-level function that does what the above code snippet does to simplify most use cases.

**Comment by dynamite-ready** [comment]
Thank you so much for your advice and comments. 

Hopefully, by the time you get a chance to read this, I will have actioned them. 

As for the comments in my code, it's just as well I didn't write the first words that popped into my head! I honestly did not expect the developer of a key dependency to read any of this. ^_^

The main aim of those comments was help out curious newbies (I'm a big fan of Knuth's 'Literate Programming' paradigm) however, it seems like I got a little carried away... But it's an honour to know you'd read and considered them. That said, please do take them with a pinch of salt, because if the API really was that hard to follow, or badly documented, I'd have not used it!

I should also mention that your library was a huge inspiration for this app. 

So many thanks. :]

**Comment by Breakthrough** [comment]
I'm glad you did write it, because feedback like that is important.  It's hard to find a stream-of-consciousness type API review over the Internet, let alone one written as well as this one.  I thought it was pretty constructive overall, and really do appreciate you sharing those thoughts.  It's an honour to know that PySceneDetect helped inspire `movie-parser`, and I'm excited to see it evolve.

Please feel free to reach out anytime on Github or Discord if you have any questions, comments, feedback, or suggestions.  When v0.6 is released, I would love to hear any feedback or improvements you might suggest so that I can take it into account for the first stable v1.0 release.  Excited to see what the future holds for `movie-parser` or any other future projects you have in mind, keep up the awesome work! 😄  🚀
